dev/sql/00-base-schema.sql

```diff
@@ -0,0 +1,76 @@
+-- Users
```

Comment:
Perhaps something indicating that it's for LSP in the filename?

Reply:
I will add a description to the `README.md` in the directory (currently it's an empty file)